mmpose/datasets/datasets/face/face_aflw_dataset.py

```diff
@@ -0,0 +1,209 @@
+
+    The landmark annotations follow the 68 points mark-up.
+
+    Args:
+        ann_file (str): Path to the annotation file.
+        img_prefix (str): Path to a directory where images are held.
+            Default: None.
+        data_cfg (dict): config
+        pipeline (list[dict | callable]): A sequence of data transforms.
+        test_mode (bool): Store True when building test or
+            validation dataset. Default: False.
+    """
+
+    def __init__(self,
+                 ann_file,
+                 img_prefix,
+                 data_cfg,
+                 pipeline,
+                 test_mode=False):
+
+        super().__init__(
+            ann_file, img_prefix, data_cfg, pipeline, test_mode=test_mode)
+
+        self.ann_info['use_different_joint_weights'] = False
+        assert self.ann_info['num_joints'] == 19
+        self.ann_info['joint_weights'] = \
+            np.ones((self.ann_info['num_joints'], 1), dtype=np.float32)
+
+        self.ann_info['flip_pairs'] = [[0, 5], [1, 4], [2, 3], [6, 11],
+                                       [7, 10], [8, 9], [12, 14], [15, 17]]
+
+        self.dataset_name = 'aflw'
+        self.db = self._get_db()
+
+        print(f'=> num_images: {self.num_images}')
+        print(f'=> load {len(self.db)} samples')
+
+    def _get_db(self):
+        """Load dataset."""
+        gt_db = []
+        for img_id in self.img_ids:
+            num_joints = self.ann_info['num_joints']
+
+            ann_ids = self.coco.getAnnIds(imgIds=img_id, iscrowd=False)
+            objs = self.coco.loadAnns(ann_ids)
+
+            rec = []
+            for obj in objs:
+                if max(obj['keypoints']) == 0:
+                    continue
+                joints_3d = np.zeros((num_joints, 3), dtype=np.float32)
+                joints_3d_visible = np.zeros((num_joints, 3), dtype=np.float32)
+
+                keypoints = np.array(obj['keypoints']).reshape(-1, 3)
+                joints_3d[:, :2] = keypoints[:, :2]
+                joints_3d_visible[:, :2] = np.minimum(1, keypoints[:, 2:3])
+
+                if 'center' in obj and 'scale' in obj:
+                    center = np.array(obj['center'])
+                    scale = np.array([obj['scale'], obj['scale']]) * 1.25
+                else:
+                    center, scale = self._xywh2cs(*obj['bbox'][:4], 1.25)
+
+                image_file = os.path.join(self.img_prefix,
+                                          self.id2name[img_id])
+                rec.append({
+                    'image_file': image_file,
+                    'center': center,
+                    'scale': scale,
+                    'rotation': 0,
+                    'joints_3d': joints_3d,
+                    'joints_3d_visible': joints_3d_visible,
+                    'dataset': self.dataset_name,
+                    'bbox': obj['bbox'],
+                    'box_size': obj['box_size'],  # normalize factor
```

Comment:
What does this comment ` # normalize factor` mean

Reply:
I want to show that `box_size` is only used as a normalize factor for evaluation.

Reply:
Okay, change it to `used as normalization factor`